userguide/aws/how-tos/add-a-certificate.adoc

```diff
@@ -54,17 +56,17 @@ In either case you will be taken to the main page of the ACM. Initially this wil
 
 The next page will give you the option to request a public or private certificate. The private option may be grayed out. Click *Request a Public Certificate*. The page that appears will ask you to specify several parameters, including one or more domain names and a validation method.
 
-image::https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ELB-request-certificate.png[align=center]
+image::../../assets/aws/ELB-request-certificate.png[align=center]
```

Comment:
@jdmooney - you only need one set of "../" to indicate the relative path: 

`image::../assets/aws/ELB-request-certificate.png[align-center]`

Reply:
The file is in userguide/aws/howtos and the images are in userguide/assets. So I  think the link is correct. And it works on my branch.